work/luyi-huang/index.html

```diff
@@ -1,6 +1,6 @@
 ---
 student: Luyi Huang
-layout: blank.liquid
+layout: coverf.gif
```

Comment:
If you want to use this as the tile image, you’d have to do image: covers.gif. Feel free to look at one of the other students’ pages for reference.